tests/LaravelRequestDocsMiddlewareTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Rakutentech\LaravelRequestDocs\Tests;
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Route;
+use Rakutentech\LaravelRequestDocs\LaravelRequestDocsMiddleware;
+
+class LaravelRequestDocsMiddlewareTest extends TestCase
+{
+    public function testMissingLRDHeader()
+    {
+        Route::get('middleware', function () {
```

Comment:
shouldn't this also have `data =>`?
Also see #178

Reply:
Should not, this is the case where `X-Request-LRD` header is not exists.
We do not mutate user's response if the request do not come from LRD UI

Reply:
But then the front won’t display the correct results? Since now the front end looks for data.

Reply:
I think this is the scenario, let see if I am missing some info:

```php
# Register middleware
Kernel.php

protected $middlewareGroups = [
    'api' => [
        \Rakutentech\LaravelRequestDocs\LaravelRequestDocsMiddleware::class,
    ];
];
```

1. Request from LRD UI 
   Send request api/v1/health 
   -> LRD UI always add X-Request-LRD header.
   -> Add data wrapper + _lrd details.

2. Request from browser/postman/curl
   Send request api/v1/health
   -> No X-Request-LRD header.
   -> Should not mutate response.

Reply:
Will reply tomorrow. Do not merge.

Reply:
Yes you are right. Good catch. 
but will check one more thing tomorrow.

Reply:
Good catch.
Ok so if `app.debug` is false, the response on LRD will become empty if this is not changed.
Some users may want to use it on prod, just don't want to share SQL, logs and Events.

![Screen Shot 2023-03-04 at 11 48 49](https://user-images.githubusercontent.com/629055/222872328-8be98934-49d0-4bbe-bca7-885a658a3446.png)

How about following?

```
        if (!config('app.debug') && $request->headers->has('X-Request-LRD')) {
            $response    = $next($request);
            $jsonContent = json_encode([
                'data' => $response->getData()
            ]);
            $response->setContent($jsonContent);
            return $response;
        }

        if (!config('app.debug')) {
            return $next($request);
        }
```

Reply:
Pushed for re-review.
**Critical**

Reply:
Probably we should provide a `config('request-docs.enabled')` as well for security reasons

Reply:
Thank you.

Yes, adding `config('request-docs.enabled')` is better.

Reply:
added